geoapps/drivers/components/data.py

```diff
@@ -103,14 +102,15 @@ def __init__(self, workspace: Workspace, params: Params, window: Dict[str, Any])
         self._initialize()
 
     def _initialize(self) -> None:
-        """ Extract data from workspace using params data. """
+        """ Extract data from the workspace using params data. """
 
         self.vector = True if self.params.inversion_type == "mvi" else False
         self.n_blocks = 3 if self.params.inversion_type == "mvi" else 1
         self.ignore_value, self.ignore_type = self.parse_ignore_values()
         self.components, self.data, self.uncertainties = self.get_data()
 
         self.locations = super().get_locations(self.params.data_object)
+        self.locations = super().z_from_topo(self.locations)
```

Comment:
Does it not always drape data on topo?!? I don't think we want that...

Reply:
Also, `z_from_topo` is used as a Params/property in other classes. This could be confusing. Maybe rename to `drape_on_topo`?

Reply:
Thank you for catching that - checking for z_from_topo now and calling a renamed set_z_from_topo method when True.